languages/elixir/exercises/concept/errors/.test

```diff
@@ -0,0 +1,2 @@
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
```

Comment:
I think this file needs to be deleted. There already is a `test_helper.exs` under its proper path.